src/types.ts

```diff
@@ -256,11 +256,28 @@ export interface MetaTransactionV1QuoteResponse extends BasePriceResponse {
     orders?: any;
 }
 
+/**
+ * Response type for /meta_transaction/v2/quote endpoint
+ */
+export interface MetaTransactionV2QuoteResponse extends BasePriceResponse {
```

Comment:
This may be a dumb question, but what's the difference between quote and price?

Reply:
No it's a great question! In general, `/quote` endpoint is intended to be called when there is an intention to trade (there are more processing we do on our end). If not, `/price` endpoint should be called if you're just loosely checking. This is a pattern we use in our other APIs as well.

In the case of meta-transaction, `/price` and `/quote` are similar but no meta-transaction object is returned & no quote report is generated in `/price` (there are some other minor differences but not huge).